### DISCUSSION TITLE
autogen.sh no longer present

### DISCUSSION THREAD

**Discussion by GeorgeAndWendy** [topaz-next/topaz#729] [Help + Q&A] [answered]
I was just making a new test server and the autogen.sh file is no longer present, so I could not build.

**Comment by zach2good** [comment, marked as answer]
I think this was resolved in Discord, but for passers-by:
A few months ago we switched to CMake as our build system on all platforms, so all other scripts have been deleted. All the build tutorials in the wiki are updated with relevant new information:

Quickstart Guide: https://github.com/topaz-next/topaz/wiki/Quick-Start-Guide
CMake: https://github.com/topaz-next/topaz/wiki/CMake-Build-Guide
More in-depth Linux guide: https://github.com/topaz-next/topaz/wiki/Server-Installation---Setup-%5BUbuntu-18.04-LTS%5D
Relevant PR: https://github.com/project-topaz/topaz/pull/1007
